systool.1

```diff
@@ -30,48 +30,48 @@ filesystem mounted.
 .SH OPTIONS
 .TP
 .B \-a
-Show attributes of the requested resource
+Show attributes of the requested resource.
 .TP
-.B \-b \fIbus
-Show information for a specific bus
+.B \-b \fIbus\fP
+Show information for a specific bus.
 .TP
-.B \-c \fIclass
-Show information for a specific class
+.B \-c \fIclass\fP
+Show information for a specific class.
 .TP
 .B \-d
-Show only devices
+Show only devices.
 .TP
 .B \-h
-Show usage
+Show usage.
 .TP
-.B \-m \fImodule_name
-Show information for a specific module
+.B \-m \fImodule_name\fP
+Show information for a specific module.
 .TP
 .B \-p
-Show absolute sysfs path to the resource
+Show absolute sysfs path to the resource.
 .TP
 .B \-v
-Show all attributes with values
+Show all attributes with value.s
```

Comment:
Looks good to me, a minor-nitpick the full-stop is misplaced.

Reply:
Oops, fixed now. :)